types/angular-translate/index.d.ts

```diff
@@ -56,6 +58,21 @@ declare module 'angular' {
             instant(translationId: string, interpolateParams?: any, interpolationId?: string, forceLanguage?: string, sanitizeStrategy?: string): string;
             instant(translationId: string[], interpolateParams?: any, interpolationId?: string, forceLanguage?: string, sanitizeStrategy?: string): { [key: string]: string };
             isPostCompilingEnabled(): boolean;
+            /**
+             * @ngdoc function
+             * @name pascalprecht.translate.$translate#negotiateLocale
+             * @methodOf pascalprecht.translate.$translate
+             *
+             * @description
+             * Returns a language key based on available languages and language aliases. If a
+             * language key cannot be resolved, returns undefined.
+             *
+             * If no or a falsy key is given, returns undefined.
+             *
+             * @param [key] Language key
```

Comment:
I think what @demensky meant was for you to remove the brackets that in JSDoc scope mean optional, but in typescript that info is already in the signature.

```suggestion
             * @param key Language key
```